### PR TITLE
Create feedstock

### PR DESCRIPTION
**Pull request by deshanadesai**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/acorns/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/deshanadesai/autodiff/archive/v0.0.1.tar.gz
+  sha256: 64c1de7798d247b3987b886b84da367d189a7debe4670c9f9d687a67c4eb7ce0
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<35]
```
Comment (scopatz) [inline recipes/acorns/meta.yaml:15]: These two lines could be a single line using  `[win or py<35]`

File: recipes/acorns/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - setuptools
```
Comment (scopatz) [inline recipes/acorns/meta.yaml:23]: You probably should remove setuptools here. It is implied by pip below

File: recipes/acorns/meta.yaml (hunk 1 above)
Comment (deshanadesai) [reply recipes/acorns/meta.yaml]: That makes sense. Edited. Thank you!

File: recipes/acorns/meta.yaml (hunk 2 above)
Comment (deshanadesai) [reply recipes/acorns/meta.yaml]: Edited. Thanks!

**Comment by scopatz** [comment]
Thanks!

File: recipes/acorns/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - pycparser
+
+test:
+  imports:
+    - src
+    - tests
```
Comment (ocefpaf) [inline recipes/acorns/meta.yaml:32]: @DeshanaDesai these are artifacts of incomplete packaging choices. I don't believe you want your package to be "import src" and having the tests outside of the module will  cause headaches.

If you are following the "src" layout for packages please take a look at PyPA recommendations about it.

Comment (deshanadesai) [reply recipes/acorns/meta.yaml:32]: Yes, completely agree. I have changed the "src" to "acorns" in the next version. But yet to be committed. Thanks for checking!

**Pull request merged**